monitoring/uss_qualifier/scenarios/scenario.py

```diff
@@ -176,6 +176,30 @@ def record_passed(self) -> None:
     def skip(self) -> None:
         self._outcome_recorded = True
 
+    def describe(self) -> str:
+        doc = self._documentation
+        severity = doc.severity or "NoSeverity"
+        url = doc.url
+        participant_str = (
+            f" for participants {', '.join(self._participants)}"
+            if getattr(self, "_participants", None)
+            else ""
+        )
+        url_str = f", doc: {url}" if url else ""
+        return f"check '{doc.name}'{participant_str} (severity {severity}{url_str})"
```

Comment:
nit: since the participant string will be, e.g., "uss1, uss2", this will produce a string like

> check 'Operational intent references can be queried by ID'uss1, uss2 (severity High https://github.com/interuss/monitoring/blob/main/monitoring/uss_qualifier/scenarios/astm/utm/dss/fragments/oir/oir_has_expected_subscription.md#-oir-is-attached-to-expected-subscription-check)

...which doesn't seem super clear or clean.

```suggestion
        return f"'{doc.name} check' ({severity} severity involving {participant_str}) documented at {url_str})"
```

...which will produce

> 'Operational intent references can be queried by ID check' (High severity involving uss1, uss2) documented at https://github.com/interuss/monitoring/blob/main/monitoring/uss_qualifier/scenarios/astm/utm/dss/fragments/oir/oir_has_expected_subscription.md#-oir-is-attached-to-expected-subscription-check

This can be adjusted in a separate PR.